ext/civicrm_admin_ui/managed/SavedSearch_Mailing_Browse_Scheduled.mgd.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+$columns = [
+  [
+    'type' => 'field',
+    'key' => 'name',
+    'dataType' => 'String',
+    'label' => E::ts('Mailing Name'),
+    'sortable' => TRUE,
+    'icons' => [],
+  ],
+  [
+    'type' => 'field',
+    'key' => 'Mailing_MailingJob_mailing_id_01.status:label',
+    'dataType' => 'String',
+    'label' => E::ts('Status'),
+    'sortable' => TRUE,
+    'icons' => [],
+    'cssRules' => [],
+  ],
+];
+
+// language only if multilingual
+if (CRM_Core_I18n::isMultilingual()) {
+  $columns[] = [
+    'type' => 'field',
+    'key' => 'language:label',
+    'dataType' => 'String',
+    'label' => E::ts('Language'),
+    'sortable' => TRUE,
+  ];
+}
+
+$columns = array_merge($columns, [
+  [
+    'type' => 'field',
+    'key' => 'created_id.display_name',
+    'dataType' => 'String',
+    'label' => E::ts('Created By'),
+    'sortable' => TRUE,
+  ],
+  [
+    'type' => 'field',
+    'key' => 'created_date',
+    'dataType' => 'Timestamp',
+    'label' => E::ts('Created Date'),
+    'sortable' => TRUE,
+  ],
+  [
+    'type' => 'field',
+    'key' => 'scheduled_id.display_name',
+    'dataType' => 'String',
+    'label' => E::ts('Send By'),
+    'sortable' => TRUE,
+  ],
+  [
+    'type' => 'field',
+    'key' => 'Mailing_MailingJob_mailing_id_01.scheduled_date',
+    'dataType' => 'Timestamp',
+    'label' => E::ts('Scheduled'),
+    'sortable' => TRUE,
+  ],
+  [
+    'type' => 'field',
+    'key' => 'Mailing_MailingJob_mailing_id_01.start_date',
+    'dataType' => 'Timestamp',
+    'label' => E::ts('Started'),
+    'sortable' => TRUE,
+  ],
+  [
+    'type' => 'field',
+    'key' => 'Mailing_MailingJob_mailing_id_01.end_date',
+    'dataType' => 'Timestamp',
+    'label' => E::ts('Completed'),
+    'sortable' => TRUE,
+  ]
```

Comment:
`civilint` is complaining about this.
```suggestion
  ],
```